suitcase/fields.py

```diff
@@ -831,6 +832,12 @@ def unpack(self, data, **kwargs):
         self._value = self.substructure()
         return self._value.unpack(data, **kwargs)
 
+    def setval(self, value):
+        if isinstance(value, suitcase.structure.Structure):
```

Comment:
So what I was thinking was:

1. Assigning a non-Structure in place of an arbitrary Structure should raise an exception, rather than overwriting it. Admittedly, there are other ways to do that.

2. A user is less likely to want to assigned directly to a larger, multi-field structure.

3. Smaller structures (like strings or RGB colors, etc.) could be more convenient to work with if the Structure subclass handled converting from a Python-native data structure to one that will be packed.

Anyway, maybe it's a little too magical, but I definitely think an exception should be thrown for overwriting a Structure with a non-Structure.